src/run/troubleshoot/analysis/ProxiesConfiguredCorrectlyAnalysis.ts

```diff
@@ -72,39 +60,19 @@ export default class ProxiesConfiguredCorrectlyAnalysis {
         }
     }
 
-    private checkDockerProxiesConfig (dkSettingsPath): AnalysisIssue | undefined {
+    private checkDockerProxiesConfig (): AnalysisIssue | undefined {
         const httpProxy = process.env.HTTPS_PROXY || process.env.https_proxy;
+        const { OverrideProxyHTTP, OverrideProxyHTTPS, ProxyHTTPMode, ProxyHttpMode } = fetchDockerSettings();
```

Comment:
My thoughts was so that rather than encounter an error which the user might percieve as a problem with `chs-dev` to output the `AnalysisOutcome` detailing the settings file is not as expected.